rpcs3/Emu/NP/np_structs_extra.cpp

```diff
@@ -217,4 +217,18 @@ namespace extra_nps
 		sceNp2.warning("ownerPrivilegeRankNum: %d", req->ownerPrivilegeRankNum);
 	}
 
+	void print_set_roommemberdata_int_req(const SceNpMatching2SetRoomMemberDataInternalRequest* req)
+	{
+		sceNp2.warning("SceNpMatching2SetRoomMemberDataInternalRequest:");
+		sceNp2.warning("roomId: %d", req->roomId);
+		sceNp2.warning("memberId: %d", req->memberId);
+		sceNp2.warning("teamId: %d", req->teamId);
+		sceNp2.warning("flagFilter: 0x%x", req->flagFilter);
+		sceNp2.warning("flagAttr: 0x%x", req->flagAttr);
+		sceNp2.warning("roomMemberBinAttrInternal: *0x%x", req->roomMemberBinAttrInternal);
+		sceNp2.warning("roomMemberBinAttrInternalNum: %d", req->roomMemberBinAttrInternalNum);
```

Comment:
these can be seperated by other random log entries. I think a single formatted entry would be better here.

Reply:
sure but this entire file does it this way